exercicios/para-casa/entregas/cintia-macchia/contaBancaria.js

```diff
@@ -0,0 +1,59 @@
+class contaBancaria {
+    constructor(nomeTitular, saldo, limite){
+        this.nomeTitular = nomeTitular,
+        this.saldo = saldo,
+        this.limite = limite
+    }
+
+    consultarSaldo(){
+        return this.saldo 
+    }
+
+    consultarSaldoTotal(saldoTotal){
+       saldoTotal = this.saldo + this.limite
+       return saldoTotal
+    }
+
+    fazerDeposito(deposito){
+        this.saldo = this.saldo + deposito
+        return this.consultarSaldo()       
+    }
+
+    fazerSaque(sacar){
+        if(sacar <= this.saldo){
+           // this.saldo = this.saldo - sacar
+           this.saldo = this.saldo - sacar
+           return this.saldo
+            
+        }else if(sacar <= this.saldo + this.limite ){
+          
+           this.saldo = this.saldo - sacar
+           
+        } else {
+           throw new Error("Saldo Insuficiente")
+
+        }
+        return this.consultarSaldo()  
+    }
+    
+    consultarLimite(){
+        return this.limite
+    }
+
+    aumentarLimite(aumentar){
+        this.limite = this.limite + aumentar
```

Comment:
Você pode usar deste jeito (se quiser e achar que faz sentido), deixando o codigo mais "limpo".
Sugestão: `this.limite += aumentar`